src/components/user/profile/header.tsx

```diff
@@ -1,5 +1,97 @@
-const Header = () => {
-  return <div className="bg-starlight-red">Header</div>;
+import { Github, Mail } from "lucide-react";
+import { FaDiscord } from "react-icons/fa";
+
+interface HeaderProps {
+  name: string;
+  email: string;
+  github: string;
+  discord: string;
+  status: string;
+  profilePic: string;
+  projects: string[];
+}
+
+const ProjectDiv = ({ text }: { text: string }) => {
+  return (
+    <p className="rounded-md bg-starlight-blue px-7 py-1.5 text-sm font-bold text-white">
+      {text}
+    </p>
+  );
+};
+
+const Projects = ({ projects }: { projects: string[] }) => {
+  return (
+    <div className="w-full overflow-x-auto">
+      <div className="flex flex-nowrap items-center gap-3">
+        {projects.map((status, index) => (
+          <ProjectDiv key={index} text={status} />
+        ))}
+      </div>
+    </div>
+  );
+};
+
+const Header = ({
+  name,
+  email,
+  github,
+  discord,
+  status,
+  profilePic,
+  projects,
+}: HeaderProps) => {
+  return (
+    <div className="mb-10 mt-7 flex max-w-full flex-row items-center gap-5 overflow-hidden">
+      <img
+        src={profilePic}
+        alt="profile picture"
+        className="hidden h-40 w-1/6 rounded-xl object-cover md:block"
+      />
+      <div className="flex max-h-full w-full max-w-full flex-col gap-1">
+        <h1 className="text-3xl font-bold">{name}</h1>
```

Comment:
using h1 directly like this messes with SEO.

should be able to use a `<p>` tag instead and i think `text-3xl` will keep it the same size.